Makefile

```diff
@@ -52,6 +52,7 @@ release:
   	  -DCMAKE_INSTALL_PREFIX="${PREFIX}"       \
   	  -DMSCORE_INSTALL_SUFFIX="${SUFFIX}"      \
   	  -DMUSESCORE_LABEL="${LABEL}"             \
+          -DUSE_SYSTEM_FREETYPE="${USE_SYSTEM_FREETYPE}" \
```

Comment:
Thank you for the review.

a) OK. Note that the file indentation around the cmake directives is messed up anyway (spaces then tabs then spaces again). You may want to normalize this.
b) Indeed. Fixed
c) This patch is merely a wrapper around the already existing cmake flag in CMakeLists.txt. There, it is specified this doesn't work on macOS or Windows. Hence the default false value. The current patch does no more, no less.

Patch updated.